ccm_web/client/src/components/UserEnrollmentForm.tsx

```diff
@@ -138,7 +138,7 @@ export default function UserEnrollmentForm (props: UserEnrollmentFormProps): JSX
     if (
       userExists === undefined ||
       role === undefined ||
-      selectedSection === undefined
+      selectedSection === undefined || firstName === undefined || lastName === undefined
```

Comment:
This might work functionally, but it's not quite logical, as `firstName` and `lastName` aren't required in all cases (and we don't know if they will be required, since `userExists` hasn't been populated).

Reply:
So this is actually a new bug. If the user does exist, this addition actually prevents the user from submitting the form, even though they have completed it and nothing on the page indicates they still need to something. I believe this needs to be changed to this, so that this condition fails if `userExists === true` and everything else (but the hidden first and last name) are finished.
```
    if (
      userExists === undefined ||
      (!userExists && firstName === undefined && lastName === undefined) ||
      role === undefined ||
      selectedSection === undefined
    ) {
      return setShowIncompleteAlerts(true)
    }
    setShowIncompleteAlerts(false)
```

Reply:
OK, I have changed the logic. I did not gave a deep thought if the logic is logical :) but it was working so there I go. 
This will help code readability!